core-developers/experts.rst

```diff
@@ -225,6 +225,7 @@ traceback             iritkatriel
 tracemalloc           vstinner
 tty                   Yhg1s*
 turtle                gregorlingl^, willingc
+turtledemo            terryjreedy*
```

Comment:
I'm not sure there's much point having two distinct entries?

We should also check that @terryjreedy is content to be listed as a *turtle* expert, as CODEOWNERS and the experts list are not bijective.

Reply:
> We should also check that @terryjreedy is content to be listed as a *turtle* expert, as CODEOWNERS and the experts list are not bijective.

He said https://github.com/python/cpython/issues/123370#issuecomment-2311954589

Reply:
Turtle is to turtledemo somewhat as tkinter is to idlelib.  Yes, I approve of this addition.